connection.py

```diff
@@ -174,18 +174,33 @@ def drop_tables(db):
     else:
         print("Tablas borradas!")
 
+def read_data_materias():
```

Comment:
Bro, I have a doubt...

If we call the method `read_data_materias()` every time we need to choice a *subject*, we'd fill the list like a million times!

So, I was thinking about declaring the list at the very begining, and calling the method `read_data_materias()` just before the first `for` loop starts.

If we do that, then, the `get_random_data_materias()` won't be needed, it'd be better to just put `subject = r.choice(listMaterias)`

What do you think?

Reply:
I agree with you!